plugins/fontconfig.lua

```diff
@@ -0,0 +1,57 @@
+-- mod-version:1 -- lite-xl 2.00
```

Comment:
Please use "lite-xl" 1.16 here to not break compatibility. Otherwise, if you want to be cutting edge just leave out the "lite-xl" part and keep only "modversion".

Reply:
This plugin targets 2.0 and above (because of new subprocess API goodness), but I can use `io.popen` (it's so stupidly simple it might still work)

Reply:
Ok, fine for version 2.0 and later so in this case I guess we should transition to modversion = 2 and coordinate with the 2.0 release.